setup.py

```diff
@@ -112,6 +112,8 @@ def run_setup(with_binary=True):
         author='GridGain Systems',
         author_email='info@gridgain.com',
         description='GridGain binary client Python API',
+        long_description=long_description,
```

Comment:
Should we put the actual description text here?

Reply:
actual description text is taken from README.md and stored to `long_description` variable